components/sensor/ags10.rst

```diff
@@ -0,0 +1,130 @@
+AGS10 Volatile Organic Compound (VOC) Sensor
+============================================
+
+.. seo::
+    :description: Instructions for setting up AGS10 VOC sensors with ESPHome
+    :image: ags10.jpg
+    :keywords: AGS10
+
+The ``ags10`` sensor platform VOC sensor allows you to use your ASAIR AGS10
+(`datasheet <http://www.aosong.com/userfiles/files/Datasheet%20AGS10.pdf>`__,
+`ASAIR`_ ) sensors with
+ESPHome. The :ref:`I²C Bus <i2c>` is
+required to be set up in your configuration for this sensor to work.
+
+.. warning::
+
+    The sensor supports up to 15kHz operation, so you should specify up to ``frequency: 15kHz`` in your ``i2c`` configuration.
+
+
+.. _ASAIR: http://www.aosong.com/en/products-86.html
+
+.. figure:: images/ags10.jpg
+    :align: center
+    :width: 30.0%
+
+    AGS10 VOC Sensor
+
+.. code-block:: yaml
+
+    # I²C Bus configuration
+    i2c:
+      sda: GPIO21
+      scl: GPIO22
+      scan: true
+      frequency: 15kHz
+
+    # Example configuration entry
+    sensor:
+      - platform: ags10
+        tvoc:
+          name: TVOC
+        address: 0x1A
+        update_interval: 60s
```

Comment:
Example YAML should be basic to allow the user to read the documentation and understand how to configure it
```suggestion
    # Example configuration entry
    sensor:
      - platform: ags10
        tvoc:
          name: TVOC
```

Reply:
Done